cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -1375,31 +1375,31 @@ end subroutine strain_rates
       
       subroutine viscous_coeffs_and_rep_pressure (strength,  tinyarea, &
                                                   Deltane,   Deltanw,  &
-                                                  Deltase,   Deltasw,  &
+                                                  Deltasw,   Deltase,  &
                                                   zetax2ne,  zetax2nw, &
-                                                  zetax2se,  zetax2sw, &
+                                                  zetax2sw,  zetax2se, &
                                                   etax2ne,   etax2nw,  &
-                                                  etax2se,   etax2sw,  &
+                                                  etax2sw,   etax2se,  &
                                                   rep_prsne, rep_prsnw,&
-                                                  rep_prsse, rep_prssw,&
+                                                  rep_prssw, rep_prsse,&
```

Comment:
In fact, I'm kind of on the fence about changing the order in the interface also. What's the justification ? it because other subroutines use this order? which ones?